### ISSUE TITLE
Issues Write Up

### ISSUE THREAD

**Issue by quinn50** [FTBTeam/FTB-Sky-Adventures#58]
<h1>Issues: Version 1.1.0</h1>
<b>General Issues </b>
- Akashic tomes created with emc crash game when opened

<b>Refined Storage Issues</b>

- 1k/64k drives obtained via EMC do not work at all, won't even go into the drives.
- 64k storage parts have a way too cheap of a EMC value, as by the time most people are going to get a RS system setup, they most likely would have a EMC farm, and not even including rewards from the quests to easily fill as many disk drives as you even would need in the early game
- More of an issue with quantum storage and RS vs the pack itself, is that the quantum storage drive being the max positive integer size, should be toned down a bit to allow 7 64k drives to be able to be in the same drive without integer overflow to -2147... etc you get the point
- Shift clicking recipe for nature compass, and the snow globe from EU2 causes a "Payload may not be larger than 32767 bytes" error disconnecting the player from the local server.

<b>Progression Issues/Suggestions</b>

<b>BIG ISSUE: Akashic when turned into draconic evolution tablet gives a 8199 emc profit(1 dirt = akashic tome)</b>
- Environmental tech crystals should have increased EMC value as you go up in tiers, maybe even only give a EMC value to the first tier. Would need testing, with only a few million EMC you can easily get max tier machines
- Quest Chest from the FTB Quests mod should either have a crafting recipe or be the reward for the "Takes money to make money!" quest, having the reward be a legendary loot box is rather redundant.
- Allow creative energy source to broken, right now it's basically bedrock
- Remove EMC value from awakened draconicum.
- Weapons, Tools, Protection & More section should have quests for thaumcraft armors, botaina armors, blood magic armor, and draconic armor. draconic sword should be the staff of power as the morning star is listed for projectE, just makes sense.
- For jetpack part, there should be an option to turn in jetpack from simply jetpack vs IC2 jetpack.
- Change energy collector mk1 recipe to not use red matter furnace change it to something like alloy smelter from EIO or induction smelter from IC2 and spit ball a emc value. Honestly if you can afford to create mk1s you probably already have a juicy emc farm by that point making them overall useless in the late game
- Increase the amount of RF required per cycle for resource fishers from IF, honestly this was the main reason I was able to effectively beat the pack fast was this.
- I would say nerf clay farm, but that would bring issues to early game.
- reduce draconic reactor core turn in by 100$
- avarita? since the pack is based off project e it could be a nice addition,  but would require alot more quest work and balancing tho

<b>Reward Store Suggestions</b>
- Replace option to buy tiny chaos fragments or remove it, it is more efficient to actually go out and kill a chaos dragon than turn in $72,900 to get one shard.
- Add botania and thaumcraft items to the reward store.
- Creative energy source quest name is listed as 15,000, when the required amount of money is 21,400.
- Add more early game things to buy, as I played I ended up just being like why buy all of these items vs just saving up quest money to unlock the repeatable quests.
- Add mechanical user as a option to buy 100-250 dollar range.
- Maybe add some store rewards you unlock as you do quests, giving a incentive to do quests.
- Make Acceleration wand a store buy only for atleast 500-1000$

**Comment by tfox83** [comment]
-EMC on RS Drives fixed in dev
-QS drive is being looked at by dev.

-(in dev) first 4 ET crystal have progressively more expensive EMC values, last two have EMC removed.
-The dev for FTBquests is adding a way to all bulk/repeat turnins without all the clicking...
-The creative energy source is meant to not be broken, this is not a change I can make.  I will add a tool tip to JEI so people know.
-During testing we went back and forth on having EMC on awakened draconicum, and since this pack is meant to be fun and sorta OP we decided to keep it.
-Some of the quests are being changed, I will take you suggestions with jetpack/armors and make a note.
-The idea of having collectors high EMC is to render them more end game, its a faily easy way to produce EMC with very little setup and no upkeep.  We do however plan to add some different alt recipes for the magic players. 
-IF Fisher energy usage has been increased in def to 512 rf/t from 100ish.
-We had loads of requests to increase the reactor core by $100, going to leave for now
-At this time we have no plans to add avarita.

-The whole reward store is being overhauled in the next few updates to have loads more items in.  I have noted your suggestions and fixed any errors. 

Thanks for this detailed report, I will label as in-progress.
